fixtures.ts

```diff
@@ -0,0 +1,48 @@
+import { test as base } from '@playwright/test';
+import { HomePage } from './pages/HomePage';
+import { LoginPage } from './pages/LoginPage';
+import { ProductPage } from './pages/ProductPage';
+import { CartPage } from './pages/CartPage';
+import { CheckoutPage } from './pages/CheckoutPage';
+
+type MyFixtures = {
+  homePage: HomePage;
+  loginPage: LoginPage;
+  productPage: ProductPage;
+  loggedInPage: HomePage;
+  cartPage: CartPage;
+  checkoutPage: CheckoutPage;
+};
+
+export const test = base.extend<MyFixtures>({
+  homePage: async ({ page }, use) => {
+    const homePage = new HomePage(page);
+    await use(homePage);
+  },
+  loginPage: async ({ page }, use) => {
+    const loginPage = new LoginPage(page);
+    await use(loginPage);
+  },
+  productPage: async ({ page }, use) => {
+    const productPage = new ProductPage(page);
+    await use(productPage);
+  },
+  loggedInPage: async ({ page }, use) => {
+    const loginPage = new LoginPage(page);
+    await page.goto(process.env.WEB_URL + '/auth/login');
```

Comment:
You already specify baseURL in https://github.com/polina-gurzhii/final-project-phurzhii/blob/main/playwright.config.ts#L30, so you do not need to use process.env.WEB_URL here and in your tests

```suggestion
    await page.goto('/auth/login');
```